app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class StaticPagesController < ApplicationController
+	def index
```

Comment:
wasn't aware of that i'll reconfigure sublime to do a two space indent... im accustomed to a four space tab...